### DISCUSSION TITLE
WAI-ARIA support

### DISCUSSION THREAD

**Discussion by TheHogginator** [segmentio/evergreen#1495] [General]
Hey folks, what's the script with WAI-ARIA support?

Noticed quite a few gaps some with workarounds some without, found this discussion that mentioned it was on the list https://github.com/segmentio/evergreen/discussions/1114#discussioncomment-795541 but that was a wee while ago is it still upcoming?  Are you open to pull requests or is this sort of thing better left for Segment internally?

**Comment by brandongregoryscott** [comment]
Hey @TheHogginator!

I wish I had a better answer, but unfortunately it's not on our immediate roadmap given our resources and internal priorities. We are certainly open to pull requests either way!
